rosdep/python.yaml

```diff
@@ -29,6 +29,10 @@ adafruit-pca9685-pip:
   ubuntu:
     pip:
       packages: [adafruit-pca9685]
+autograd-pip:
+  '*':
```

Comment:
Unfortunately, wildcard matching is not valid for the os only for the version.

Reply:
Ah, thanks - I was wondering exactly that. I have amended the PR.

Reply:
Yeah, that would match the `*` distribution ;-)

Thanks for the update.